cartographer_grpc/framework/rpc.cc

```diff
@@ -49,11 +49,11 @@ Rpc::Rpc(int method_index,
       execution_context_(execution_context),
       rpc_handler_info_(rpc_handler_info),
       service_(service),
-      new_connection_event_{Event::NEW_CONNECTION, this, false},
-      read_event_{Event::READ, this, false},
-      write_event_{Event::WRITE, this, false},
-      finish_event_{Event::FINISH, this, false},
-      done_event_{Event::DONE, this, false},
+      new_connection_event_pending_(false),
```

Comment:
maybe defining these defaults in the header is slightly more readable?

Reply:
Done.